### PR TITLE
Asset Cache page cleanup.

### PR DESCRIPTION
**Pull request by AMZN-stankowi**
Creating new branch with fixed sign-off for https://github.com/o3de/o3de.org/pull/1480

Signed-off-by: AMZN-stankowi <4838196+AMZN-stankowi@users.noreply.github.com>

### Submission Checklist:

* [x] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [x] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [x] **Consistency** - Does the content consistently follow the [style guide](https://o3de.org/docs/contributing/to-docs/style-guide/)?
* [x] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

File: content/docs/user-guide/assets/pipeline/asset-cache.md [hunk 1]
```diff
@@ -23,20 +23,25 @@ There are three parts to the Asset Cache: the **Asset Database**, the **Asset Ca
 
 The Asset Database is only used by Asset Processor. The Asset Database is a development time database containing details about assets, jobs, **Asset Builders**, and scan directories. It contains all the information Asset Processor requires to determine which assets have been processed, and which assets need to be processed.
 
-With the information in the Asset Database, Asset Processor can be suspended in the middle of a process job and resume when relaunched, remembering all the pending process jobs in the job queue.
+With the information in the Asset Database, Asset Processor can be suspended in the middle of a process job and resume when relaunched, rebuilding the queue of jobs to process.
```
Comment (lmbr-pip) [inline content/docs/user-guide/assets/pipeline/asset-cache.md:26]: "rebuilding" here sounds like re-processing existing assets. Is that your intent?

File: content/docs/user-guide/assets/pipeline/asset-cache.md [hunk 2]
```diff
@@ -23,20 +23,25 @@ There are three parts to the Asset Cache: the **Asset Database**, the **Asset Ca
...
 
 ### Asset Catalog
 
 The Asset Catalog is a runtime catalog of product assets and product dependencies. The Asset Catalog contains a subset of information from the Asset Database; the information about assets required at runtime. **Asset Browser** uses the Asset Catalog to find the runtime product assets on disk that you request for your projects. 
 
+The information in the Asset Catalog XML file in the cache is not the same information that is available in the runtime project. During development, **O3DE Editor** and game launchers use the `assetcatalog.xml` file for tracking assets. In bundled release builds, the only assets available in the catalog are those that are specifically mounted within [asset bundles.](/docs/user-guide/packaging/asset-bundler/) Each bundle contains a Delta Catalog, which is a subset of the Asset Catalog that contains information about the assets within the asset bundle. As asset bundles are mounted, the Delta Catalog XML file is loaded, and the in-memory Asset Catalog is expanded to contain the information from the Delta Catalog.
```
Comment (lmbr-pip) [inline content/docs/user-guide/assets/pipeline/asset-cache.md:32]: nit: Each bundle contains a Delta Catalog, which is a subset of the Asset Catalog, containing information about the assets within the asset bundle. ?

File: content/docs/user-guide/assets/pipeline/asset-cache.md [hunk 3]
```diff
@@ -23,20 +23,25 @@ There are three parts to the Asset Cache: the **Asset Database**, the **Asset Ca
...
+
 ### Asset Cache
 
- The Asset Cache contains the Asset Database, the Asset Catalog, and the product assets generated by Asset Builders.
+The Asset Cache contains the Asset Database, the Asset Catalog, and the product assets generated by Asset Builders.
+
+The Asset Cache contains versions of the product asset for the host platform as well as versions for each target platform. For example, when you create a project for Windows and mobile platforms, the cache contains separate versions of the Asset Cache for Windows and each mobile platform. If you are developing the project on Windows and deploying on Windows, the same product assets are used for the host and target platforms.
```
Comment (lmbr-pip) [inline content/docs/user-guide/assets/pipeline/asset-cache.md:38]: nit: The Asset Cache contains versions of the product asset for the host platform*,* as well as versions for each target platform

File: content/docs/user-guide/assets/pipeline/asset-cache.md [hunk 4]
```diff
@@ -23,20 +23,25 @@ There are three parts to the Asset Cache: the **Asset Database**, the **Asset Ca
...
+
+In the `Cache` directory of your project, there are subdirectories for each platform supported by your project. In the platform directories, there are directories that correspond to every scan directory in which Asset Processor found source assets. The scan directories contain the product assets generated by Asset processor.
 
- The Asset Cache contains versions of the product asset for the host platform as well as versions for each target platform. For example, when you create a project for Windows and mobile platforms, the cache contains separate versions of the Asset Cache for Windows and each mobile platform. If you are developing the project on Windows and deploying on Windows, the same product assets are used for the host and target platforms.
+Asset Processor assumes it is the only process that makes changes to the Asset Cache. If you modify or delete files in the Asset Cache, including deleting product asset files while Asset Processor is not running. the changes might not be noticed by Asset Processor.
```
Comment (lmbr-pip) [inline content/docs/user-guide/assets/pipeline/asset-cache.md:42]: nit: "while Assets Process is not running, then the changes.."

File: content/docs/user-guide/assets/pipeline/asset-cache.md [hunk 5]
```diff
@@ -23,20 +23,25 @@ There are three parts to the Asset Cache: the **Asset Database**, the **Asset Ca
...
 
- 
- In the `Cache` directory of your project, there are subdirectories for each platform supported by your project. In the platform directories, there are directories that correspond to every scan directory in which Asset Processor found source assets. The scan directories contain the product assets generated by Asset processor.
+The Asset Cache is used during development, and by default the Editor and game launchers in development builds load assets from the asset cache. The default behavior for release builds is to use [bundled assets.](/docs/user-guide/packaging/asset-bundler/)
```
Comment (lmbr-pip) [inline content/docs/user-guide/assets/pipeline/asset-cache.md:44]: nit: The Asset Cache is intended to only be used only during development. By default the Editor and game launchers in non-release builds load assets from the asset cache.

There is not a notion of "development" build, so be a little careful with this term. Users have debug, profile or release builds, which they can use in their development "phase", we also have a "development branch" for O3DE.

**Review by lmbr-pip — COMMENTED** [review]
Some minor suggestions but it looks great.

**Review by amzn-mike — APPROVED** [review]

**Review by micronAMZN — APPROVED** [review]

File: content/docs/user-guide/assets/pipeline/asset-cache.md (hunk 1 above)
Comment (AMZN-stankowi) [reply content/docs/user-guide/assets/pipeline/asset-cache.md]: I've re-worded this.

File: content/docs/user-guide/assets/pipeline/asset-cache.md (hunk 2 above)
Comment (AMZN-stankowi) [reply content/docs/user-guide/assets/pipeline/asset-cache.md]: Fixed

File: content/docs/user-guide/assets/pipeline/asset-cache.md (hunk 3 above)
Comment (AMZN-stankowi) [reply content/docs/user-guide/assets/pipeline/asset-cache.md]: Fixed

File: content/docs/user-guide/assets/pipeline/asset-cache.md (hunk 4 above)
Comment (AMZN-stankowi) [reply content/docs/user-guide/assets/pipeline/asset-cache.md]: Fixed

File: content/docs/user-guide/assets/pipeline/asset-cache.md [hunk 6]
```diff
@@ -23,20 +23,25 @@ There are three parts to the Asset Cache: the **Asset Database**, the **Asset Ca
 
 The Asset Database is only used by Asset Processor. The Asset Database is a development time database containing details about assets, jobs, **Asset Builders**, and scan directories. It contains all the information Asset Processor requires to determine which assets have been processed, and which assets need to be processed.
 
-With the information in the Asset Database, Asset Processor can be suspended in the middle of a process job and resume when relaunched, remembering all the pending process jobs in the job queue.
+With the information in the Asset Database, Asset Processor can be closed when it has not finished processing all assets. Next time it is launched, the Asset Processor will use this information to figure out what assets it has already been processed.
```
Comment (willihay) [inline content/docs/user-guide/assets/pipeline/asset-cache.md:26]:
```suggestion
With the information in the Asset Database, Asset Processor can be closed when it has not finished processing all assets. Next time it is launched, the Asset Processor will use this information to figure out what assets it has already processed.
```

File: content/docs/user-guide/assets/pipeline/asset-cache.md [hunk 7]
```diff
@@ -23,20 +23,25 @@ There are three parts to the Asset Cache: the **Asset Database**, the **Asset Ca
...
+Asset Processor assumes it is the only process that makes changes to the Asset Cache. If you modify or delete files in the Asset Cache, including deleting product asset files while Asset Processor is not running, the changes might not be noticed by Asset Processor.
 
- 
- In the `Cache` directory of your project, there are subdirectories for each platform supported by your project. In the platform directories, there are directories that correspond to every scan directory in which Asset Processor found source assets. The scan directories contain the product assets generated by Asset processor.
+The Asset Cache is used during development, and by default the Editor and game launchers in development builds load assets from the asset cache. The default behavior for release builds is to use [bundled assets.](/docs/user-guide/packaging/asset-bundler/)
```
Comment (willihay) [inline content/docs/user-guide/assets/pipeline/asset-cache.md:44]:
```suggestion
The Asset Cache is used during development, and by default the Editor and game launchers in development builds load assets from the asset cache. The default behavior for release builds is to use [bundled assets](/docs/user-guide/packaging/asset-bundler/).
```

**Review by willihay — APPROVED** [review]

**Pull request merged**